extensions/plugin-activity-depth-changed/src/activityDepthChangedPlugin.tsx

```diff
@@ -0,0 +1,49 @@
+import type { Activity } from "@stackflow/core";
+import type { StackflowReactPlugin } from "@stackflow/react";
```

Comment:
```suggestion
import type { StackflowPlugin } from "@stackflow/core";
```

- 추후에 `@stackflow/vue` 같은것이 나오더라도 사용가능한 플러그인 같습니다 ㅋㅋ
- 따라서, React 관련한 인터페이스를 안쓴다면, core에 존재하는 플러그인 인터페이스를 쓰셔도 좋아요.